src/bindings/html/head.js

```diff
@@ -1,5 +1,7 @@
 'use strict';
 
+import './polyfill';
```

Comment:
This gets transpiled to `const {} = getModule('bindings/html/polyfill');`.  @zbraniecki, can you help me fix this in our babel bundler?